homeassistant/components/sensor/torque.py

```diff
@@ -0,0 +1,118 @@
+"""
+homeassistant.components.sensor.torque
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Get data from the Torque OBD application.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.torque/
+"""
+
+import re
+from homeassistant.helpers.entity import Entity
+
+
+DOMAIN = 'torque'
+DEPENDENCIES = ['http']
+SENSOR_EMAIL_FIELD = 'eml'
+DEFAULT_NAME = 'vehicle'
+HTTP_RESPONSE = 'OK'
+HTTP_VERB = 'GET'
+ENTITY_NAME_FORMAT = '{0} {1}'
+
+API_PATH = r'/api/torque'
+SENSOR_NAME_KEY = r'userFullName(\w+)'
+SENSOR_UNIT_KEY = r'userUnit(\w+)'
+SENSOR_VALUE_KEY = r'k(\w+)'
+
+PATH_MATCH = re.compile(API_PATH)
```

Comment:
Home Assistant HTTP allows either a string or a regex for url matching. Since your link is hard coded, no need to compile it.